src/systemctl/systemctl.c

```diff
@@ -4632,11 +4649,11 @@ static int show_one(
                  * 4: program or service status is unknown
                  */
                 if (info.pid_file && access(info.pid_file, F_OK) == 0)
```

Comment:
it appears the table above is the wrong one really... we should never have returned any of these codes... aaaaah, what a confusion... not sure why LSB defined two tables here in the first place...